kubernetes/package.dhall

```diff
@@ -14,4 +14,5 @@
 , Compute = ./Compute.dhall
 , Space = ./Space.dhall
 , Memory = ./Memory.dhall
+, VPASettings = ./VPASettings.dhall
```

Comment:
so instead of `kubernetes/VPASettings.dhall` I would do `kubernetes/autoscaler/vertical-pod-autoscaler/settings.dhall` (which matches the upstream repository setup)